schema.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$id": "https://raw.githubusercontent.com/DeterminateSystems/bootspec/v1.0.0/schema.json",
```

Comment:
This probably won't work, since this schema didn't exist when we released v1.0.0, back in 2023 :)